### PR TITLE
Pay 4548

### PR DESCRIPTION
**Pull request by arbab-cog**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: api/src/functionalTest/resources/application-functional-tests.properties [hunk 1]
```diff
@@ -1,4 +1,5 @@
-test.url=${TEST_URL:http://localhost:8000}
+#test.url=${TEST_URL:http://localhost:8000}
+test.url=${TEST_URL:http://payment-api-pr-925.service.core-compute-preview.internal}
```
Comment (aravindan89) [inline api/src/functionalTest/resources/application-functional-tests.properties:2]: Please put back the old URL.

File: api/src/functionalTest/resources/application-functional-tests.properties [hunk 2]
```diff
@@ -35,7 +35,7 @@ pci-pal.antenna.grant.type=${PCI_PAL_ANTENNA_GRANT_TYPE:client_credentials}
 pci-pal.antenna.tenant.name=${PCI_PAL_ANTENNA_TENANT_NAME:HMCTS}
 pci-pal.antenna.user.name=${PCI_PAL_ANTENNA_USER_NAME:APIUser}
 pci-pal.antenna.client.id=${PCI_PAL_ANTENNA_CLIENT_ID:HMCTSStage}
-pci-pal.antenna.client.secret=${PCI_PAL_ANTENNA_CLIENT_SECRET:Vm82kOUZAf9U4Kms3hIQM03jqsDQKc8aHIGy5I9y}
+pci-pal.antenna.client.secret=${PCI_PAL_ANTENNA_CLIENT_SECRET:dummy}
```
Comment (pats-john) [inline api/src/functionalTest/resources/application-functional-tests.properties:38]: As requested by Anil...

File: api/src/functionalTest/resources/application-functional-tests.properties (hunk 1 above)
Comment (pats-john) [reply api/src/functionalTest/resources/application-functional-tests.properties]: Reverted...

File: api/src/functionalTest/resources/application-functional-tests.properties (hunk 2 above)
Comment (pats-john) [reply api/src/functionalTest/resources/application-functional-tests.properties:38]: Resolved.

**Review by aravindan89 — APPROVED** [review]

**Review by AnilDhawaleGitHub — APPROVED** [review]

**Review by YasharRahvar — APPROVED** [review]

**Pull request merged**